types/jest-in-case/jest-in-case-tests.ts

```diff
@@ -11,8 +11,8 @@ function subtract(minuend: number, subtrahend: number) {
 }
 
 beforeEach(() => {
-    jest.spyOn(global, 'describe').mockImplementation((title, fn) => fn());
-    jest.spyOn(global, 'test').mockImplementation((name, fn) => fn());
+    jest.spyOn(global, 'describe').mockImplementation((title, fn) => jest.fn());
```

Comment:
Don't know why I did that... 

```fn```  should be cast to a function because it is unknown

```
 jest.spyOn(global, 'describe').mockImplementation((title, fn) => (fn as () => void)());
```